tests/lib/bluetooth/ble_conn_params/src/unity_test.c

```diff
@@ -0,0 +1,1160 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <errno.h>
+#include <unity.h>
+#include <stdint.h>
+#include <string.h>
+#include <stddef.h>
+#include <bm/bluetooth/ble_conn_params.h>
+#include <bm/softdevice_handler/nrf_sdh.h>
+#include <zephyr/sys/util.h>
```

Comment:
Is this header used?

Can add `zephyr/sys/iterable_sections.h` here.

Reply:
Removed, and added.